scripts/run_e2e_tests.py

```diff
@@ -112,19 +112,19 @@
     'classroompagefileuploadfeatures': RERUN_POLICY_NEVER,
     'collections': RERUN_POLICY_NEVER,
     'contributordashboard': RERUN_POLICY_KNOWN_FLAKES,
-    'coreeditorandplayerfeatures': RERUN_POLICY_KNOWN_FLAKES,
-    'creatordashboard': RERUN_POLICY_KNOWN_FLAKES,
-    'embedding': RERUN_POLICY_KNOWN_FLAKES,
+    'coreeditorandplayerfeatures': RERUN_POLICY_NEVER,
+    'creatordashboard': RERUN_POLICY_NEVER,
+    'embedding': RERUN_POLICY_NEVER,
     'explorationfeedbacktab': RERUN_POLICY_NEVER,
     'explorationhistorytab': RERUN_POLICY_KNOWN_FLAKES,
     'explorationimprovementstab': RERUN_POLICY_ALWAYS,
-    'explorationstatisticstab': RERUN_POLICY_KNOWN_FLAKES,
-    'explorationtranslationtab': RERUN_POLICY_KNOWN_FLAKES,
+    'explorationstatisticstab': RERUN_POLICY_NEVER,
+    'explorationtranslationtab': RERUN_POLICY_NEVER,
     'extensions': RERUN_POLICY_NEVER,
     'featuregating': RERUN_POLICY_ALWAYS,
     'fileuploadextensions': RERUN_POLICY_NEVER,
     'fileuploadfeatures': RERUN_POLICY_KNOWN_FLAKES,
-    'learner': RERUN_POLICY_NEVER,
+    'learner': RERUN_POLICY_KNOWN_FLAKES,
```

Comment:
Is this an expected change? i.e, moving policy in the negative direction?

Reply:
Yes, there has been a slight regression in this suite so we will be temporarily loosening the policy.